native/python/pyjp_object.cpp

```diff
@@ -256,17 +286,32 @@ static PyObject* PyJPException_expandStacktrace(PyObject* self)
 	JP_PY_CATCH(NULL);
 }
 
+PyObject *PyJPException_args(PyBaseExceptionObject *self)
+{
+	if (self->args == NULL)
+		Py_RETURN_NONE;
+	Py_INCREF(self->args);
+	return self->args;
+}
+
 static PyMethodDef exceptionMethods[] = {
 	{"_expandStacktrace", (PyCFunction) PyJPException_expandStacktrace, METH_NOARGS, ""},
 	{NULL},
 };
 
+static PyGetSetDef exceptionGetSets[] = {
+	{"_args", (getter) PyJPException_args, NULL, ""},
+	{0}
+};
+
```

Comment:
We can't implement args directly as it has a conflict with the descriptor in Pythons chain, but we can do the work here an just redirect back to the Python wrapper layer.